### PR TITLE
CotE - Ikoma Orator

### PR DESCRIPTION
**Pull request by SArendeli**
Added implementation and tests for Ikoma Orator

### CODE REVIEW

**Comment by SArendeli** [comment]
Le sigh...I'll fix this

File: test/server/cards/06-CotE/IkomaOrator.spec.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                    player2: {
+                        inPlay: ['bayushi-liar'],
+                        honor: 11
+                    }
+                });
+                this.ikomaOrator = this.player1.findCardByName('ikoma-orator');
+                this.bayushiLiar = this.player2.findCardByName('bayushi-liar');
+                this.shamefulDisplay = this.player2.findCardByName('shameful-display', 'province 1');
+                this.assassination = this.player1.clickCard('assassination');
+                this.banzai = this.player1.clickCard('banzai');
```
Comment (jamesoddy) [inline test/server/cards/06-CotE/IkomaOrator.spec.js]: You would normally use the `findCardByName` function not the `clickCard` function if you are just after a reference to the card.

File: test/server/cards/06-CotE/IkomaOrator.spec.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+                this.noMoreActions();
+
+                this.initiateConflict({
+                    type: 'political',
+                    province: this.shamefulDisplay,
```
Comment (jamesoddy) [inline test/server/cards/06-CotE/IkomaOrator.spec.js]: Not a problem doing this, but you do not usually need to specify the province with the `this.initiateConflict` function.  Just to let you know.

**Comment by jamesoddy** [comment]
It would be helpful if you mentioned the issue in your PR description.  LIke this:

Closes #2714

File: test/server/cards/06-CotE/IkomaOrator.spec.js (hunk 2 above)
Comment (SArendeli) [reply test/server/cards/06-CotE/IkomaOrator.spec.js]: Good to know, I removed the Shameful definition and this line as well.

**Comment by SArendeli** [comment]
Gotcha - I'll add the Issue # going forward.

**Pull request merged**